google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/CopyJobConfiguration.java

```diff
@@ -131,6 +138,20 @@ public Builder setWriteDisposition(JobInfo.WriteDisposition writeDisposition) {
       return this;
     }
 
+    /**
+     * The labels associated with this job. You can use these to organize and group your jobs. Label
+     * keys and values can be no longer than 63 characters, can only contain lowercase letters,
+     * numeric characters, underscores and dashes. International characters are allowed. Label
+     * values are optional. Label keys must start with a letter and each label in the list must have
+     * a different key.
+     *
+     * @param labels labels or {@code null} for none
```

Comment:
Remove the repetition of labels.

Reply:
![Screenshot from 2019-09-27 14-38-04](https://user-images.githubusercontent.com/44407134/65757303-7fc9ee80-e134-11e9-9c90-04c8ad1bf845.png)
screenshot from proto file

Reply:
ok